dts/bindings/sensor/st,stm32-temp-common.yaml

```diff
@@ -13,3 +13,9 @@ properties:
       type: int
       default: 3300
       description: Temperature sensor voltage in millivolts
+
+    adc-calib:
```

Comment:
I don't have a great proposal to make but it doesn't seem this burden should be put on ADC clients but rather on ADC device themselves (otherwise this bit will have to be duplicated on all clients which feels wrong to me).